mmv1/third_party/terraform/services/compute/data_source_google_compute_network.go.tmpl

```diff
@@ -24,9 +24,12 @@ func DataSourceGoogleComputeNetwork() *schema.Resource {
 				Computed: true,
 			},
 
-			// TODO: this should eventually be TypeInt, but leaving as
-			// string for now to match the resource and to avoid a
-			// breaking change.
+			"network_id": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+
+			// Deprecated in favor of network_id
 			"numeric_id": {
```

Comment:
Also add a `Deprecated` flag for the field in the datasource following the handwritten section of https://googlecloudplatform.github.io/magic-modules/breaking-changes/make-a-breaking-change/#field-deprecation-due-to-removal-or-rename